manifests/board-image/0.20240720.0-matrix.bot.toml

```diff
@@ -0,0 +1,34 @@
+format = "v1"
+[[distfiles]]
+name = "root-lpi4a-20240720_171951.ext4.zst"
```

Comment:
What's this file?

Reply:
Sorry due to repo cache in local, the bot uploads some unexpected file. It should been fixed.